manifests/g/GoogleContainerTools/Skaffold/2.13.0/GoogleContainerTools.Skaffold.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GoogleContainerTools.Skaffold

```

```suggestion
PackageIdentifier: Google.ContainerTools.Skaffold
```
After updating the package identifier, you will need to change the folder and the name of the manifests